docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
     entrypoint: /var/www/time_tracker/docker-entrypoint.sh
     volumes:
       - .:/var/www/time_tracker
-      - ./extension:/var/www/time_tracker_extension/
+      - ~/Projects/time_tracker_extension:/var/www/time_tracker_extension/
```

Comment:
I have another folder for projects. Maybe there is a way to make it more flexible? 
Maybe you could use relative path?

Reply:
added ability to set project folder for gem, you can put EXTENSION_GEM_PATH into specific file and enable this by command

Reply:
Thanks.